instrumentation/jaxrs-client/jaxrs-client-2.0/jaxrs-client-2.0-jersey-2.0/javaagent/src/main/java/org/glassfish/jersey/client/ResponseCallbackWrapper.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.glassfish.jersey.client;
+
+import static io.opentelemetry.javaagent.instrumentation.jaxrsclient.v2_0.JerseyClientUtil.handleException;
+
+import javax.ws.rs.ProcessingException;
+import org.glassfish.jersey.process.internal.RequestScope;
+
+// implemented interface is package private so wrapper needs to be in the same package
```

Comment:
What about module system and split packages? Or we are agent and ignore all that?

Reply:
I believe that split packages are not an issue, these are verified when module is defined, defining extra classes shouldn't really change anything in that regard. Sealed packages and singed jars could have issues with this though, but it seems really unlikely that someone would sign jersey jars.

Reply:
we do this in one other place that I'm aware of

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/d1ff1e12acdc6c239faea5e2100fb514c46618c2/instrumentation/rxjava/rxjava-1.0/library/src/main/java/rx/__OpenTelemetryTracingUtil.java#L6-L16